content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+
+.. math::
+    k_{ground} \simeq (1-i) \sqrt{ \frac{\omega \mu \sigma}{2} }
+    :label: kwavenumber_steadystate
+
+In the air, the conductivity is almost 0
+
+.. math::
+    k_{air} \simeq \omega \sqrt{ \mu_0 \varepsilon_0}
+    :label: kwavenumber_air
+
+Taking the problem from the point of view of the electric field, we know the equation :eq:`E_wave_propagation_equation` has a solution in the form of:
+
+.. math::
+    E_x (z) = U e^{i k z} + D e^{-i k z}
+    :label: Electric field components
+    
+.. math::
+    H_y (z) = \frac{(\nabla \times \mathbf{E_x})_y}{- i \omega \mu} = \frac{k}{ \omega \mu} (D e^{-i k z} - U e^{i k z}) = \frac{1}{Z} (D e^{-i k z} - U e^{i k z})
+    :label: Magnetic field components
+
+with :math:`\mathbf{E_x} = E_x \mathbf{\hat{x}}`  and U and D are the complex amplitudes of the Up and Down components of the field and Z the intrinsic impedance of the space.
+
+Writing the solution in the j-th layer (See :numref:`MTlayeredEarth`), we got:
+
+ .. math::
+    E_{x,j} (z) = U_j e^{i k (z-z_{j-1})} + D_j e^{-i k (z-z_{j-1})}
+    :label: Electric field components in layers
+    
+ .. math::
+    H_{y,j} (z) = \frac{1}{Z_j} (D_j e^{-i k (z-z_{j-1})} - U_j e^{i k (z-z_{j-1})})
+    :label: Magnetic field components in layers
+
+
+Which can be rewrite as:
+
+ .. math::
+    \left(\begin{matrix} E_{x,j} \\ H_{y,j} \end{matrix} \right) = \left(\begin{matrix} 1 & 1 \\ -\frac{1}{Z_j} & \frac{1}{Z_j} \end{matrix} \right) \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+    = P_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right)
+    :label: Propagation matrix 
+
+The transition of the Up and Down component inside a layer can then be write as such
+
+ .. figure:: images/InsideLayer.png
+    :align: center
+    :scale: 100% 
+    :name: InsideLayer
+
+    Transition inside a layer, variables definition.
+
+
+.. math::
+    \left(\begin{matrix} U_j' \\ D_j' \end{matrix} \right)  = \left(\begin{matrix} e^{i k h_j} & 0 \\ 0 & e^{-i k h_j} \end{matrix} \right) \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+    = T_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+
+With the variables U, D, U' and D' define as in (:numref:`InsideLayer`)
+
+Using the continuity of the tangential \\(\\mathbf{E_x}\\) and \\(\\mathbf{H_y}\\) field at the interfaces, we find an iterative relation between the fields in consecutive layers:
+
+.. math::
+    \left(\begin{matrix} E_{x,j} \\ H_{y,j} \end{matrix} \right) = P_j T_j P^{-1}_J \left(\begin{matrix} E_{x,j+1} \\ H_{y,j+1} \end{matrix} \right)
+
+We are now only missing a Boundary Condition to be able to compute our MT forward modeling. A reasonable one is to set the Down Amplitude to 1 and the Up Amplitude to 0 in the last layer, as there is no reflection from an other interface below.
+
+.. math::
+    \left(\begin{matrix} U_n \\ D_n \end{matrix} \right)  = \left(\begin{matrix} 0 \\ 1 \end{matrix} \right) 
+
```

Comment:
this is a pretty abrupt end to this section. What are the implications of having set the boundary conditions? Here, I think we need an explanation of how the forward simulation is now solved (just in words)